utils/makefile.python

```diff
@@ -112,15 +114,19 @@ quiet_cmd_pyinstall = INSTALL   $2
       cmd_pyinstall = $(PIP) $(PIP_VERBOSE) install $(PIP_INST) -e $2$(PY_SETUP_EXTRAS)
 
 # $2 path to folder with setup.py, this uses pip from pyenv (not OS!)
-quiet_cmd_pyenvinstall = PYENV     install $2
+quiet_cmd_pyenvinstall = PYENV     [pyenvinstall] $(PY_ENV): install $2
       cmd_pyenvinstall = \
-	if ! cat $(PY_ENV)/requirements.sha256 2>/dev/null | sha256sum --check --status 2>/dev/null; then \
-		rm -f $(PY_ENV)/requirements.sha256; \
-		$(PY_ENV_BIN)/python -m pip $(PIP_VERBOSE) install -e $2$(PY_SETUP_EXTRAS) &&\
-		sha256sum requirements*.txt > $(PY_ENV)/requirements.sha256 ;\
+	_IMP_STATEMENT="import $(foreach var,$(wildcard $(PYOBJECTS)),$(var), ) pkg_resources"; \
+	_PRINT="print('          [pyenvinstall] pkg_resources.working_set --> resources %s' % len([True for i in pkg_resources.working_set]))"; \
+	if ! $(PY_ENV_BIN)/python -c \
+		"import sys; sys.path.pop(0); $$_IMP_STATEMENT; $$_PRINT" 2>/dev/null;\
```

Comment:
I end up to understand, but why display this information ?

Reply:
> I end up to understand, but why display this information ?

good question .. I will remove it.

Reply:
> I end up to understand, but why display this information ?

Line has been removed ..